marlowe-playground-client/src/JavascriptEditor/View.purs

```diff
@@ -82,8 +84,13 @@ jsEditor state = slot _jsEditorSlot unit component unit (Just <<< HandleEditorMe
       mContents <- LocalStorage.getItem StaticData.jsBufferLocalStorageKey
       let
         contents = fromMaybe JSE.escrow mContents
+
+        decoratedContent = joinWith "\n" [ decorationHeaderString, contents, decorationFooterString ]
+
+        numLines = Array.length $ lines decoratedContent
       model <- Monaco.getModel editor
-      Monaco.setValue model contents
+      Monaco.setValue model decoratedContent
+      pure unit
 
   component = monacoComponent $ JSM.settings setup
```

Comment:
while we're here, I think `component` should be moved into `JavascriptEditor.State` as `mkComponent` and then `JavascriptEditor.View` just contains `jsEditor state = slot _jsEditorSlot unit mkEditor unit (Just <<< HandleEditorMessage)` or something